### ISSUE TITLE
Intro behaviour

### ISSUE THREAD

**Issue by nomadics9** [jellyfin/jellyfin-androidtv#3159]
1.Install intro plugin.
2.Go to androidtv with default settings
3.Start any video
4.No intro

---workaround---

1.Go to settings
2.Playback
3.Resume pre-roll
4.Select 1 minute 
5.Intro works

Any option less than one minute doesn't run the pre roll even if playing media for the first time.

I think by default it should be 1 minute. Kinda defeats the purpose if I have to tell the user to change this setting to 1 minute to have the full experience.

Thank you for your awesome work🫡

### Application version

0.15.12

### Device information

Android tv

### Android version

Android 10

### Jellyfin server version

Latest

**Comment by nielsvanvelzen** [comment]
We don't have an official intro plugin so I'm not sure which specific plugin you mean. Can you link a repository/website for it?

**Comment by nomadics9** [comment]
https://raw.githubusercontent.com/BrianCArnold/jellyfin-plugin-intros/master/manifest.json

im using this plugin. but the options are native to the app. as I do not face the same issue in web nor jellyfin app in phones

**Comment by nielsvanvelzen** [comment]
Right now we only support intros for movies in The Android TV app. This behavior is in the old playback code which is hard to work with. I'm working on rewriting all of that and fixing the behavior of intros is a part of that (e.g. play the intro before starting any set of items instead of movies).

Closing as duplicate of #2787.

**Comment by nomadics9** [comment]
Update:

It just works now. I'm not sure why, but I did change the format for the intro to webm instead of mp4.

Thank you again for your amazing work and much appreciation for your response.
